src/app/[lang]/(future)/tools/metronome/page.tsx

```diff
@@ -1,18 +1,11 @@
 'use client';
 
-import { Add, Remove } from '@mui/icons-material';
-import {
-    Box,
-    Button,
-    IconButton,
-    Paper,
-    Slider,
-    TextField,
-    Typography,
-} from '@mui/material';
+import { Center, Column } from '@/components/shared/Layout';
```

Comment:
Use the "@shared" import alias. 

To do so add the Layout to the index exports file in the "shared" directory